### PR TITLE
make switch to bz2 tarball

### PR DESCRIPTION
**Pull request by diizzyy**
Maintainer: @xypron 

Description:

Switch to bz2 tarball

Signed-off-by: Daniel Engberg daniel.engberg.lists@pyret.net

### CODE REVIEW

**Comment by xypron** [comment]
@diizzyy 
The commit message does not explain why we should change to an xz tarball.

Regards

Heinrich Schuchardt

File: devel/make/Makefile [hunk 1]
```diff
@@ -9,11 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=make
 PKG_VERSION:=4.2.1
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```
Comment (yousong) [inline devel/make/Makefile]: Minor note: switching tarball format is not expect to cause change in final package content, so there is no need to bump `PKG_RELEASE` here.

**Comment by yousong** [comment]
> @diizzyy 
> The commit message does not explain why we should change to an xz tarball.

Actually it's switching to `bz2` and there is no `xz` tarball provided at the moment ;)  I guess the motivation behind this change is that `bz2` tarball of make should be a few hundreds KB smaller than the `gz` counterpart.

**Comment by diizzyy** [comment]
Yes, it's to save bandwidth and space.

**Pull request merged**